lib/loader/loader.c

```diff
@@ -283,7 +283,7 @@ struct evmc_instance* evmc_load_and_configure(const char* config,
 
         char* kv_sep = strchr(option, '=');
         const char* name = option;
-        char* value = NULL;
+        char* value = "";  // Set value to empty to handle the "...,option,..." case.
```

Comment:
Do you want to create an issue so this is changed in the next ABI version?

Reply:
After considered this, I like it this way. Not passing null pointers to VMs is nice. Otherwise, all VM implementations would need to check for null values. Also, I don't want to encourage the `,option=,` being interpreted differently from `,option,`. Also, you can see that the number of test cases has decreased.

I will squash the two "loader:" commits, but put it separately for now because consequences are quite interesting.